src/components/settings/FilterSettings.svelte

```diff
@@ -42,14 +43,29 @@
   items={languageNameValues}
 />
 <CheckOption name="Show moderator messages" store={showModMessage} />
-<MultiDropdown
-  name="Blocked users"
-  store={channelFilters}
-  getDisplayName={(n, v) => v.name}
-  getBool={n => channelFilters.get(n).blacklist}
-  setBool={(n, v) =>
-    channelFilters.set(n, { ...channelFilters.get(n), blacklist: v })}
-/>
+<Subheader style="height: 1.5rem; margin-top: 1.5rem;">
+  Blocked Users
+</Subheader>
+<Row>
+  <Col style="padding-top:0px;">
+    <MultiDropdown
+      name="YouTube chat"
+      store={channelFilters}
+      getDisplayName={(n, v) => v.name}
+      getBool={n => channelFilters.get(n).blacklist}
+      setBool={(n, v) =>
+        channelFilters.set(n, { ...channelFilters.get(n), blacklist: v })}
+    />
+  </Col>
+  {#if $enableMchadTLs}
+    <Col style="padding-top:0px">
+      <MultiDropdown
+        name="MChad"
+        store={mchadUsers}
+      />
+    </Col>
+  {/if}
+</Row>
```

Comment:
If the dropdown is too small (or if the name is too long) you can't remove the blocked user cause you can't click the `X` button:

![image](https://user-images.githubusercontent.com/20059164/127812046-873c8d4c-1bd7-472c-8834-f1f9277cec6d.png)

MIght want to make it grow/wrap or separate the dropdowns to 2 rows.

Reply:
Or maybe we can just move the `on:Click` to trigger on the whole item instead of just the `X` button?

Reply:
> Or maybe we can just move the `on:Click` to trigger on the whole item instead of just the `X` button?

This isn't clear enough. I'm just going to have the focused one expand to take up the full width.

Reply:
Ok, I did the grow one.